deploy.py

```diff
@@ -1057,6 +1060,13 @@ def main(args):
       dest="features",
       help=("Compiles the OpenSK application with support for nfc."),
   )
+  main_parser.add_argument(
+      "--configure",
+      action="store_true",
+      default=False,
```

Comment:
If this default is `False` and the target is opensk, can you please print the warning from above?
```
      info("Your device is not yet configured, and lacks some functionality. "
           "If you run into issues, this command might help:\n\n"
           "./tools/configure.py \\\n"
           "    --certificate=crypto_data/opensk_cert.pem \\\n"
           "    --private-key=crypto_data/opensk.key\n\n"
           "Please read the Certificate considerations in docs/customization.md"
           " to understand the privacy trade-off.")
```

Also we'd have to update the `docs/` to add --configure to the deploy command, or explain that you might want to configure.

Reply:
Actually, let's keep configure defaulted to true to keep existing behavior.

Defaulting to False was a mistake.